service/src/test/java/bio/terra/workspace/service/admin/AdminServiceTest.java

```diff
@@ -34,9 +34,11 @@
 import java.util.Map;
 import java.util.UUID;
 import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.Tag;
 import org.junit.jupiter.api.Test;
 import org.springframework.beans.factory.annotation.Autowired;
 
+@Tag("connectedPlus")
 public class AdminServiceTest extends BaseConnectedTest {
```

Comment:
can we have BaseConnectedPlustTest and BaseConnectedTest instead of setting tag on each test?

Reply:
I don't want to do that. It is a lot easier to manage inclusion by setting the tag on each test, rather than messing with the class hierarchy. I think the class structure should be controlling methods and mocks, not test suites.